packages/bpk-mixins/src/mixins/_buttons.scss

```diff
@@ -35,8 +35,6 @@
   margin: 0;
   padding: $bpk-button-padding-y $bpk-button-padding-x;
   border: 0;
-  background-image: linear-gradient(-180deg, $bpk-button-gradient-start-color 0%, $bpk-button-gradient-end-color 100%);
```

Comment:
This is actually wrong if we want to avoid the breaking change in the mixins, we need to keep the primary style here and remove